specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -2005,6 +2008,25 @@
         }
       }
     },
+    "ElasticVersionListFormat": {
+      "description": "Elastic Version List Format",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ElasticVersionListProperties"
+        }
+      }
+    },
+    "ElasticVersionListProperties": {
+      "description": "Elastic Version Properties",
+      "type": "object",
+      "properties": {
+        "version": {
+          "description": "Available elastic version of the given region",
+          "type": "string"
+        }
+      }
+    },
```

Comment:
Can you explain what all this extra structure is needed?  From what I can see, the information content of each list item is exactly the same -- a version string.  Why is it necessary to bury that two levels deep in an object?

Reply:
Hey, for list/get apis rpaas has suggested to use value and nextLink where value is an array of objects where in to define any parameter or property other than id, name and type needs to be inside properties object otherwise rpaas throws an invalid response payload error. I did tried that in the previous pr but it created issues. 

Here is the link to the doc: https://armwiki.azurewebsites.net/rpaas/rpaasextensionapi.html#listcollection-get:~:text=forwarded%20to%20ARM.-,Response%20Body,-The%20whole%20payload